docs/DeveloperWorkflow/flint_performance.rst

```diff
@@ -1,4 +1,29 @@
 .. _DeveloperWorkflow:
 
 FLINT Performance
-#################
+
+FLINT uses scientific modulesmodels to estimate fluxes of GHGs between pools. 
+There are several models available for each type of GHG flux e.g. Tier 1 
+formulas or emission factors and more sophisticated Tier 3 models for forest 
+growth or soil carbon dynamics. The user can start with a simple model 
+for all land uses and can then switch to more accurate models for the most 
+important land cover types when better data becomes available. More importantly,
+these models allow FLINT to make projections of GHG fluxes. The model only needs
+to receive the activity data to run its calculations formulae whether those
+describe that is in the past or in the future.
+
+FLINT is using the same models for making projections as for monitoring 
+implementation progress. This ensures that the approach used for planning 
+is the same as the one used for reporting, so the results are consistent and
+time series are seamless at the transition from reporting to projections.
+In many other cases the tool used for planning is different from the tool used
+for reporting and it is hard to identify whether the discrepancy between the
+two is due to implementation or to measurement errors.
+
+FLINT can apply accurate models for project areas that have extensive data, while
+it can apply less demanding models for the rest of the countries. This ensures 
+that achievements reached by projects are not lost when they are included into national 
+accounts. Most countries are currently using systems that are not spatially explicit 
+like FLINT. In these systems, land cover types are aggregated and the specific achievements 
+of a project area are reduced to the average emission applied to that land cover type or 
+they cannot be identified at all within the national accounts.
```

Comment:
I think this documentation is incomplete :eyes: 

@aornugent Can we please have your inputs over the same?